bundles/org.openhab.binding.guntamatic/src/main/java/org/openhab/binding/guntamatic/internal/GuntamaticHandler.java

```diff
@@ -483,6 +503,7 @@ public void initialize() {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Invalid hostname configuration");
         } else {
             updateStatus(ThingStatus.UNKNOWN);
+            logger.debug("scheduleWithFixedDelay(pollGuntamatic, 1, {}, TimeUnit.SECONDS)", config.refreshInterval);
```

Comment:
Don't see why you want to log the called line like this.
```suggestion
```

Reply:
I added this for debugging (long init time) before i recognized that the delay comes from the 120s issue. Can be removed again...

Reply:
removed